spec/indexers/hyrax/active_fedora_file_set_indexer_spec.rb

```diff
@@ -0,0 +1,151 @@
+# frozen_string_literal: true
+RSpec.describe Hyrax::ActiveFedoraFileSetIndexer do
+  include Hyrax::FactoryHelpers
+
+  let(:file_set) do
```

Comment:
This is a straight up move of the code that was in `file_set_indexer_spec.rb` _(which used to test the indexer for AF `::FileSet`, but now tests indexer for Valkyrie `Hyrax::FileSet`)_.